tests/test_well/test_wells.py

```diff
@@ -1,124 +1,84 @@
+def test_wells_copy_same_names(testwells):
+    assert testwells.copy().names == testwells.names
 
-    mywell_list = loadwells1
 
-    mywells = Wells()
-    mywells.wells = mywell_list
+def test_wells_setter_same_names(testwells):
+    wells2 = Wells()
+    wells2.wells = testwells.wells
+    assert wells2.names == testwells.names
 
-    assert "WELL33" in mywells.names
 
+def test_well_names(testwells):
+    for well in testwells.wells:
+        assert well.name in testwells.names
+        assert testwells.get_well(well.name) is well
 
-def test_get_dataframe_allwells(loadwells1):
-    """Get a single dataframe for all wells"""
-
-    mywell_list = loadwells1
-
-    mywells = Wells()
-    mywells.wells = mywell_list
 
-    df = mywells.get_dataframe(filled=True)
-
-    #    assert df.iat[95610, 4] == 345.4128
+def test_get_dataframe_allwells(testwells):
+    """Get a single dataframe for all wells"""
 
-    logger.debug(df)
+    logger.debug(testwells.get_dataframe(filled=True))
 
 
 @tsetup.plotskipifroxar
-def test_quickplot_wells(tmpdir, loadwells1):
+def test_quickplot_wells(tmpdir, testwells, generate_plot):
     """Import wells from file to Wells and quick plot."""
+    if not generate_plot:
+        pytest.skip()
+    testwells.quickplot(filename=join(tmpdir, "quickwells.png"))
 
-    mywell_list = loadwells1
 
-    mywells = Wells()
-    mywells.wells = mywell_list
-    mywells.quickplot(filename=join(tmpdir, "quickwells.png"))
-
-
-def test_wellintersections(tmpdir, loadwells1):
+def test_wellintersections(tmpdir, testwells):
     """Find well crossing"""
 
-    mywell_list = loadwells1
-
-    mywells = Wells()
-    mywells.wells = mywell_list
-    dfr = mywells.wellintersections()
-    logger.info(dfr)
+    dfr = testwells.wellintersections()
     dfr.to_csv(join(tmpdir, "wells_crossings.csv"))
 
 
-def test_wellintersections_tvdrange_nowfilter(loadwells1):
+def test_wellintersections_tvdrange_nowfilter(tmpdir, testwells):
     """Find well crossing using coarser sampling to Fence"""
+    testwells.limit_tvd(1300, 1400)
+    testwells.downsample(interval=6)
 
-    mywell_list = loadwells1
-
-    mywells = Wells()
-    mywells.wells = mywell_list
-    print("Limit TVD and downsample...")
-    mywells.limit_tvd(1300, 1400)
-    mywells.downsample(interval=6)
-    print("Limit TVD and downsample...DONE")
-
-    dfr = mywells.wellintersections()
-    print(dfr)
-
-
-def test_wellintersections_tvdrange_no_wfilter(loadwells1):
-    """Find well crossing using coarser sampling to Fence, no
-    wfilter settings.
-    """
-
-    mywell_list = loadwells1
-
-    mywells = Wells()
-    mywells.wells = mywell_list
-    print("Limit TVD and downsample...")
-    mywells.limit_tvd(1300, 1400)
-    mywells.downsample(interval=6)
-    print("Limit TVD and downsample...DONE")
-
-    dfr = mywells.wellintersections()
-    print(dfr)
+    dfr = testwells.wellintersections()
+    dfr.to_csv(join(tmpdir, "wells_crossings.csv"))
```

Comment:
I know there wasnt an assert here previously, but would be good to add one now, even as a snapshot.

Reply:
Also goes for the other tests in here you touch if it is not too much work.

Reply:
Added snapshot tests for all of them :)